code/modules/telescience/telescope/telescopeDialogues.dm

```diff
@@ -252,6 +252,34 @@
 			else
 				return 0
 
+/* WATCHFUL EYE SENSOR DIALOGUE */
+
+/datum/dialogueMaster/telescopeEye
+	dialogueName = "Watchful Eye Sensor"
+	start = /datum/dialogueNode/telEyeStart
+	visibleDialogue = 0
+
+/datum/dialogueNode
+	telEyeStart
+		nodeImage = "valiantStatic.png" // placeholder until i upload a png
```

Comment:
?

Reply:
i did actually forget to upload the png lol. it's fixed accordingly